Assets/Scripts/BulletManagement/Patterns/SemiCircle.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using UnityEngine;
+
+namespace BulletManagement.Patterns
+{
+    /*
+     * Emits concentric rings of bullets that subtend a specified angle
+     */
+    public class SemiCircle : BulletPattern
+    {
+        // The angle that subtends the bullet arc
+        public float arcAngle = 150f;
+        // The angle to rotate the pattern (e.g. 270 = straight down)
+        public int offsetAngle = 270;
```

Comment:
Is there a reason why the `offsetAngle` is in integers while the arc angle is in float?

Reply:
Lolwhoops. Do you feel like having fractional angles is valuable, or does 1 degree give us enough granularity? My preference would be to change `arcAngle` to `int`.

Reply:
Up to you. I prefer float because it makes sense in context of angles and there is less to worry about in terms of casting.

Reply:
Actually, let's just use float to keep it standard.